src/stream/src/task/barrier_manager/managed_state.rs

```diff
@@ -92,41 +157,46 @@ impl ManagedBarrierState {
             create_mview_progress: Default::default(),
             state_store,
             streaming_metrics,
+            await_epoch_completed_futures: FuturesOrdered::new(),
         }
     }
 
     /// Notify if we have collected barriers from all actor ids. The state must be `Issued`.
-    fn may_notify(&mut self, prev_epoch: u64) {
+    fn may_have_collected_all(&mut self, prev_epoch: u64) {
```

Comment:
Can we update the docstring and naming of this function? `may_have_collected_all` seems kind of vague, I didn't really get the use of it.

From the code it seems to call `sync_epoch` and update the mview progress mainly.

Reply:
I've added docstring on the method. The naming `may_xxx` is to be consistent with the original method to help better compare with the original logic for code review.